README.md

```diff
@@ -45,9 +45,9 @@ Tip: you can skip version (`@53.0`) or/and OS name (`:Windows 10`).
 
 ### Chrome
 
-For adding `chromeOptions` you can use `BROWSERSTACK_CHROME_OPTIONS=XXX` where `XXX` stand for example for `--disable-plugins` or if you need to set multiple options use `BROWSERSTACK_CHROME_OPTIONS="--disable-plugins --autoplay-policy=no-user-gesture-required"`
+For adding `chromeOptions` you can use `BROWSERSTACK_CHROME_ARGS=XXX` where `XXX` stand for example for `--disable-plugins` or if you need to set multiple options use `BROWSERSTACK_CHROME_ARGS="--disable-plugins --autoplay-policy=no-user-gesture-required"`
 BrowserStack supports the [full complement of ChromeOptions](https://sites.google.com/a/chromium.org/chromedriver/capabilities).
-
+This option works only when `BROWSERSTACK_USE_AUTOMATE` is se to `1`.
```

Comment:
According to me, A better user experience would be to automatically set `BROWSERSTACK_USE_AUTOMATE` to `1`,  if `BROWSERSTACK_CHROME_ARGS` is present. You could also print some kind of a message along the lines of:
*Using BrowserStack Automate since BROWSERSTACK_CHROME_ARGS was passed.*

But this can lead to extra logic to deal with cases when someones **does not** want to use Automate but passes `BROWSERSTACK_CHROME_ARGS` while running tests.

The reason I am in favour of automatically assuming 'Automate' in this scenario is because soon there will be a lot of "capabilities" which will show up here which are specific to Automate. The documentation needs to be refactored in a way to clearly display distinction between Automate/JS-Testing Args (and pass `BROWSERSTACK_USE_AUTOMATE` if we go that way)

@AndreyBelym  what do you think ?